src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -150,6 +150,7 @@ var HTMLDOMPropertyConfig = {
     sizes: MUST_USE_ATTRIBUTE,
     span: HAS_POSITIVE_NUMERIC_VALUE,
     spellCheck: null,
+    summary: MUST_USE_ATTRIBUTE,
```

Comment:
I just did a quick test - looks like this can be set with a property or attribute, so let's make this `null` to retain option value (`MUST_USE_*` denotes that only 1 way of setting the value is possible so we can't use the other even if it's faster).

PS, thanks for jumping on this quickly :)